index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment:
Huh, the tutorial is still going well. My suggestion is to use more emojis to get the player more stoked!
```suggestion
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
```suggestion
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```